src/functions/menus/filterMenusByLocation.js

```diff
@@ -1,28 +1,27 @@
-import formatHeirarchialMenu from "./formatHeirarchialMenu";
+import { camelCase } from "lodash";
 
+import formatHierarchicalMenu from "@/pesayetu/functions/menus/formatHeirarchialMenu";
 /**
  * Filter menus array by menu location.
  *
- * @param  {Array}  menus     The array of WP menus to filter.
- * @param  {Array}  locations The array of locations for filtering.
- * @return {object}           An object containing the requested locations as individual objects.
+ * @author WebDevStudios
+ * @param {Array} menus     The array of WP menus to filter.
+ * @param {Array} locations The array of locations for filtering.
+ * @return {object}         An object containing the requested locations as individual objects.
  */
 export default function filterMenusByLocation(menus, locations) {
   const data = {};
-
   // Loop each menu location.
   locations.forEach((location) => {
     // Convert dashes to underscores.
     const locationName = location.replace(/-/g, "_");
-
     // Filter menus array by location and assign to new object.
-    const wpmenu = menus.filter((menu) => {
+    const wpmenu = menus?.filter(function filterMenu(menu) {
```

Comment:
`(menu) => {` should be fine. Was there any issue or this is just a style change?